lib/libzutil/zutil_nicenum.c

```diff
@@ -180,5 +184,9 @@ zfs_niceraw(uint64_t num, char *buf, size_t buflen)
 void
 zfs_nicebytes(uint64_t num, char *buf, size_t buflen)
 {
-	zfs_nicenum_format(num, buf, buflen, ZFS_NICENUM_BYTES);
+	if (getenv("ZFS_KB_IS_1000") != NULL) {
```

Comment:
You intruduce a new environment variable, but there is no documentation about it.